contrib/mesos/pkg/scheduler/components/framework/framework.go

```diff
@@ -128,14 +123,6 @@ func New(config Config) Framework {
 					return false
 				}
 
-				// the executor IDs must not identify a kubelet-executor with a group that doesn't match ours
```

Comment:
I'd like to see a docs change in the known issues as part of this PR -- this change leads to a particularly nasty surprise. The docs change can be short and to the point.